README.md

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
What's up with this change?

Reply:
running 'format' from root, cause checks failed without it.

Reply:
Can we split this into a separate PR? That seems totally unrelated to re-creating the source map.

Reply:
https://github.com/WebKit/Speedometer/pull/387